src/main/java/app/backend/controller/optimization/OptimizationController.java

```diff
@@ -168,7 +168,16 @@ public ResponseEntity<Void> orderOptimization(
             return ResponseEntity
                     .status(BAD_REQUEST)
                     .build();
-        } catch (HttpServerErrorException | JsonProcessingException exception) {
+        } catch (HttpServerErrorException exception) {
+            if (exception.getStatusCode().value() == 515) {
```

Comment:
co to za error code?

Reply:
brak wyników optymalizacji bo user dał za mało czasu, customowy kod, chyba że znasz jakiś co pasuje

Reply:
niby nic a niby wszystko pasuje więc słabo :(

może [to](https://pawelpluta.com/why-should-you-return-http-422/)?

Reply:
ok, zmienioned na 422